source/content/guides/solr-drupal/02-solr-drupal.md

```diff
@@ -194,8 +194,11 @@ Click **Search API** to return to the Search API overview page located in `admin
 Click on the server’s name to view the server. The **View** tab displays server connection information, schema version, and indices.
 To save and post the schema information, select the **Pantheon Search Admin** tab, and click **Post Solr Schema** to send your custom schema to the Solr 8 server.
 
+To access the **Pantheon Search Admin** tab, you must be logged in as a SuperAdmin or in the (user/1) account.
+
```

Comment:
That is an incorrect statement. You have to be logged in with someone that has the "Execute Pantheon Search admin task" permission. Adding that permission to any role will allow all users in that role access the tab.